src/Spaces.ts

```diff
@@ -1,15 +1,32 @@
-import { Types } from 'ably';
+import { Types, Realtime } from 'ably';
 import SpaceOptions from './options/SpaceOptions';
 import Space from './Space';
-
 class Spaces {
   private spaces: Record<string, Space>;
   private channel: Types.RealtimeChannelPromise;
   ably: Types.RealtimePromise;
 
-  constructor(ably: Types.RealtimePromise) {
+  readonly version = '0.0.1';
```

Comment:
Should this be read from `package.json` ?

Reply:
When used in the browser we can't guarantee that package.json will be available

Reply:
Wouldn't that be statically inserted during the build?

Reply:
Not without some specific build stage to inject that variable

Reply:
Isn't that what the build task does?

Reply:
The build task just compiles the typescript, similar to react-hooks which [also uses a version string](https://github.com/ably-labs/react-hooks/blob/main/src/AblyReactHooks.ts#L7)

Reply:
I see, didn't realize you can't do this with TypeScript compiler. I'd suggest we compile with Vite, but that seems somewhat overkill. Not ideal to have this not synced automatically however, an easy source of bugs.